include/osquery/http_client.h

```diff
@@ -0,0 +1,438 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#pragma once
+
+#include <boost/asio.hpp>
+#include <boost/asio/deadline_timer.hpp>
+#include <boost/asio/ssl.hpp>
+#include <boost/beast/core.hpp>
+#include <boost/beast/http.hpp>
+#include <boost/optional/optional.hpp>
```

Comment:
I would try to include only what you need instead of including `<boost/asio.hpp>` and `<boost/beast/core.hpp>`, otherwise you might find compilation times growing uncomfortably.

Reply:
These two headers are required to compile the code successfully.